### PR TITLE
Add download endpoint for compliance checker reports

### PR DESCRIPTION
**Pull request by Bobfrat**
screen shot 2017-09-17 at 10 07 41 pm

### CODE REVIEW

**Comment by Bobfrat** [comment]
this is for:
https://github.com/ioos/compliance-checker/issues/478

**Comment by jbosch-noaa** [comment]
Excellent!  This will be a valuable add that many folks have been asking for. :)

File: cchecker_web/processing.py [hunk 1]
```diff
@@ -60,6 +91,33 @@ def compliance_check(job_id, dataset, checker):
...
+    :param int limit: The degree of strictness, 1 being the strictest, and going up from there.
+    :param str checker: The name of the compliance checker test
+    '''
+    points = aggregates['scored_points']
+    out_of = aggregates['possible_points']
+    source_name = aggregates['source_name']
+
+    # Print out the headers, (slightly customized from typical CC report)
+    print('\n')
+    print("-" * 80)
```
Comment (benjwadams) [inline cchecker_web/processing.py]: Could we refactor this at some point?  There are libraries which will abstract away printing of columns to make text tables.

Comment (Bobfrat) [reply cchecker_web/processing.py]: yeah this is pretty ugly. I grabbed it (and slightly modified it) from here: https://github.com/ioos/compliance-checker/blob/5bd2afe7380eb5e3c22fed6b314154fb5a9dd07a/compliance_checker/suite.py#L324-L328

**Pull request merged**